wcfsetup/install/files/lib/system/form/builder/field/UploadFormField.class.php

```diff
@@ -345,13 +345,22 @@ public function getFieldHtml()
     public function updatedObject(array $data, IStorableObject $object, $loadValues = true)
     {
         if ($loadValues) {
-            // first check, whether an getter for the field exists
-            if (\method_exists($object, 'get' . \ucfirst($this->getObjectProperty()) . 'UploadFileLocations')) {
+            $expectUploadFile = false;
+
+            // first check, whether a getter for the field exists; preferring get{$fieldName}UploadFiles() before get{$fieldName}UploadFileLocations()
+            if (\method_exists($object, 'get' . \ucfirst($this->getObjectProperty()) . 'UploadFiles')) {
```

Comment:
This code is in order to deserve a dedicated temporary variable for `\ucfirst($this->getObjectProperty())` to then be able to:
```suggestion
            if (\method_exists($object, "get{$fieldName}UploadFiles")) {
```
making this much more readable.